lib/membrane/rtcp/packet.ex

```diff
@@ -73,15 +74,38 @@ defmodule Membrane.RTCP.Packet do
     with {:ok, %{header: header, body_size: length, padding?: padding?}} <-
            Header.parse(raw_header),
          <<body::binary-size(length), rest::binary>> <- body_and_rest,
-         {:ok, {body, _padding}} <- RTP.Utils.strip_padding(body, padding?),
-         {:ok, packet_module} <- BiMap.fetch(@packet_type_module, header.packet_type),
-         {:ok, packet} <- packet_module.decode(body, header.packet_specific) do
-      do_parse(rest, [packet | acc])
+         {:ok, {body, _padding}} <- RTP.Utils.strip_padding(body, padding?) do
+      case parse_packet(body, header) do
+        {:ok, packet} ->
+          do_parse(rest, [packet | acc])
+
+        {:error, :unknown_packet_type} = error ->
+          Membrane.Logger.debug("""
+          Ignoring rtcp packet with packet type #{header.packet_type}:
+          #{inspect(raw_header <> body_and_rest, limit: :infinity)}
+          Reason: #{inspect(error)}
```

Comment:
I think we should keep the reason as the error's tuple reason as we do in other places. Also we should decide if handling of such packets should have log level `debug` or `warn`, from what I see we mostly use warn for invalid packets.

Reply:
One of the issues is that BiMap returns `:error` when it does not find a match, which can be pretty confusing. I tried to write it so that this specific error is caught and made more clear, but actual errors in parsing are passed through.

Reply:
Sure, that is not the problem. I just meant using `Reason: :unkown_packet_type` instead of `Reason: {:error, :unkown_packet_type}`

Reply:
Whoops. Didn't mean to mark your comments are resolved. Still new to the Github PR workflow.